installer/compose/quickstart/upload.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+if [ -z "$SP_INITIAL_ADMIN_EMAIL" ]; then
+  USERNAME="admin@streampipes.apache.org"
+else
+  USERNAME="$SP_INITIAL_ADMIN_EMAIL"
+fi
+
+if [ -z "$SP_INITIAL_ADMIN_PASSWORD" ]; then
+  PASSWORD="admin"
+else
+  PASSWORD="$SP_INITIAL_ADMIN_PASSWORD"
+fi
+
+while true; do
+  if curl -s "http://backend:8030/streampipes-backend/api/v2/auth/login" --max-time 3 &> /dev/null; then
+    break
```

Comment:
can we echo something in this case?
In addition, `break` will only stop the loop right? But in this case, we might want to stop the execution of the whole script. If we do so, we can increase `--max-time` as well to 10 or so

Reply:
Hi @bossenti , thank you for your feedback! Certainly, we can implement a confirmation mechanism within this script. Once the script verifies that the StreamPipes backend is ready and the login API is accessible, it can exit the loop and proceed to the next step. At this point, we can display a message saying, "StreamPipes backend is now ready." Does this approach sound suitable to you?

Regarding exiting the loop, you're correct that using `break` will only terminate the loop, allowing the script to continue with subsequent steps. This is indeed our intention, especially after confirming the readiness of the Streampipes backend under the specified condition. So I think it's better to keep `break` and not stop the whole script.

The `--max-time` option in the curl command is designated to limit the execution time, in seconds, encompassing the duration required to resolve the host, establish a connection, and transfer data. It serves as a precaution against indefinite or excessively long waits due to network complications or an unresponsive target host. Setting this parameter to 10 seconds is a feasible adjustment. I will change it to 10.

Reply:
Ah sorry, I put my comment at the wrong statement...

My only concern was: If we only wait 9s (3 seconds with max-time 3) the backend might not yet be available in some cases. In this case we shouldn't continue with the script  because this would probably lead to errors later (for sure if the backend doesn't get ready in time).
So me asking for termination was only thinking about this special edge case where the backend does not get ready within the specified waiting period.
But when we extend the waiting period to 30 seconds (3 seconds with max-time 10) this should not occur

Reply:
Hi, @bossenti ! Thanks for the suggestion, I will change the `max-time` to 10 in next commit.

Reply:
Hi, @bossenti ! I have changed the `max-time` in latest commit.